.git-blame-ignore-revs

```diff
@@ -9,3 +9,17 @@
 
 # SOLR-14920 [google java formatter applied]
 d17a1cdcebb46a6c9d360a6569319dd66021d346
+fce7554ecde2ae556da97eb4b7d8bd4eb2299f71
+077214042aeeb460e8e689eab2217ed34ce778fc
+2c31e74de4b09592aa58f001487dad2f309f692f
+8078824a21a29fc01f627d3d408da1769c9b863b
+b4f585d299a908b4243e0c7e297c4e3495752b84
+42a6f1b93a178c7957c7a1afea2417e646e982af
+c1d582675a536496ad811f071f2dc7a13bdb1b63
+475246256403135e2d5b2c7605fc1ba1da631d1f
+ab7a30c1b089a95d276d540535bfb9959d229c27
+610c835b16f19b8157cfd490931f76762f570fe2
+4ef3765c10ae1c3a782c6cd989fdb80c2905689a
+9f70c367f4350700a4c92065bef3cae4ed370c1b
+bb872f74f715fa5f161833d91a7491e8b72c6c18
+
```

Comment:
```suggestion
8081b216e6e17f1827ff4d6102cb408c3c599cfe
406350aab7fa3623f33da096fbcf0dcdc79d8c83
```
These two are for JWT on branch_9x 
SOLR-15907 Apply spotless only on JWT classes (#616)

Reply:
Thanks! Added in chronological order to the list